test.sh

```diff
@@ -781,3 +781,52 @@ no_more_events=$(curl -sSfg -u $SK: $HOST/v1/events?starting_after=$last_event \
 zero_events=$(curl -sSfg -u $SK: $HOST/v1/events?starting_after=$last_event \
                  | grep -oE '^  "data": \[\]')
 [ -n "$zero_events" ]
+
+# create a pending payout
```

Comment:
Would you be OK to remove comments? Code looks self explanatory to me (here and all comments below).